src/braket/experimental/autoqasm/README.md

```diff
@@ -56,21 +56,27 @@ def bell_state() -> None:
 You can view the output format, which is OpenQASM, by running `bell_state().to_ir()`.
 
 AutoQASM enables users to use more complicated program constructs with a compact and readable
-structure. We can demonstrate this with an active reset program:
+structure. We can demonstrate this with a program that conditionally prepares multiple Bell states:
 ```
-# A program that actively resets a qubit back to the ground state
+# A program that is conditioned on the measurement result of qubit 0 to prepare two
+# Bell states on qubit pairs (1, 2) and (3, 4).
```

Comment:
The reset example is replaced because (1) the conversation that reset might not be the easiest to understand for a getting-started example, and (2) the example implies reset is something user needs to implement while a device often already has its own reset instruction.